### PR TITLE
(#182) Updated tests for testing CTS API v2 update

### LINKED ISSUE #182: Update CTS Tests after CTS API v2 is live
Some of the tests will fail when  CTS API v2 update is live. We need to update the tests listed below:

> **ESTIMATE** TBD

### What's the expected change?
1. **Name of the test** :  Analytics.Feature
- [ ] Change all Prop 10 values to be "NCI" in place of "National Cancer Institute"

2. **Name of the test** :  Analytics.js
- [ ] Change the locator 'div.pager__num' to ".pager__navigation"
- [ ] Change wait time from 2000 to 4000.

3. **Name of the test** :  BasicSearchPageTest.feature
**Scenario: Negative: User is searching for a keyword that does not exist on tablet**
- [ ] Remove the space in front of "Your" so that "No available options found.  Your search will be based on the text above." will be replaced by "No available options found. Your search will be based on the text above."

 **Scenario: Negative: User is not able to search for an incorrect zip code on Mobile**
- [ ] And "U.S. Zip Code" form section is displayed" is replaced with the following:  And "U.S. ZIP Code" form section is displayed
- [ ] Then alert "Please enter a valid 5 digit U.S. zip code" is displayed in "U.S. Zip Code" section is replaced by Then alert "Please enter a valid 5 digit U.S. ZIP code" is displayed in "U.S. Zip Code" section

4.  **Name of the test** :  BasicSearchPageTest.feature/index.js
- [ ] Add wait time of 500ms after the step "cy.get('.results-page h1').should('have.text', 'Clinical Trials Search Results');"

5. **Name of the test** : LocationSection.feature
- [ ]  Add "And browser waits" after every time search is executed and wherever necessary

6. **Name of the test** : CancerTypeSection.feature
- [ ] Add the following step in different places to accommodate test failures due to performance:
  "And browser waits"

- [ ] Add the following statement to verify that user is on Search Results page after selecting different combinations of Types, Subtypes, Stages, Findings
And trial info displays "Results 1-1  of \d+ for your search.*" 

7. **Name of the test** : CancerTypeAutosuggest.js**
- [ ] Add code to verify that user is on Search Results page after selecting different combinations of Types, Subtypes, Stages, Findings.

8. **Name of the test** : SearchResultsPageTests.feature**
- [ ] Add the following step in different places to accommodate test failures due to performance:
  "And browser waits"
- [ ] Updated the scenario: :"Scenario: User is able to print results"

9. **Name of the test** : TrialDescriptionGeneralPageComponents.feature**
- [ ]  Add the following step in different places to accommodate test failures due to performance:
  "And browser waits"

10. Increase wait time from 2000ms to 4000ms in cypress/integration/common/index.js

### What's the current functionality?
-

### What's the updated acceptance criteria?
```gherkin
```

### Additional details / screenshot
- Screenshot

### Related Tickets
* Original Story Ticket #9999
* Original Story Ticket #9999

### PR DESCRIPTION
**Pull request by alinai**
Closes #182 

@KateMashkinaNIH , please review.

### CODE REVIEW

File: cypress/integration/CTS/Analytics/Analytics.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ When('user checks {string} checkbox', (label) => {
 });
 
 And('browser waits', () => {
-  cy.wait(2000);
+  cy.wait(4000);
```
Comment (kate-mashkina) [inline cypress/integration/CTS/Analytics/Analytics.js:55]: please revert this change

File: cypress/integration/CTS/BasicSearchPageTest/index.js [hunk 2]
```diff
@@ -66,6 +70,7 @@ And('user clicks on {string} button', (buttonLabel) => {
 Then('the search is executed and results page is displayed', () => {
     cy.location('pathname').should('contain', 'search/r');
     cy.get('.results-page h1').should('have.text', 'Clinical Trials Search Results');
+    cy.wait(500);
```
Comment (kate-mashkina) [inline cypress/integration/CTS/BasicSearchPageTest/index.js:73]: remove this

File: cypress/integration/CTS/CancerTypeSection/cancerTypeSubtypeStageSideEffectTest.js [hunk 3]
```diff
@@ -9,10 +9,8 @@ fieldAttribute.set('Side Effects/Biomarkers/Participant Attributes', 'fin');
 
 
 Then("field with placeholder appears", dataTable => {
-
-    for (const { field, placeholder } of dataTable.hashes()) {
+     for (const { field, placeholder } of dataTable.hashes()) {
```
Comment (kate-mashkina) [inline cypress/integration/CTS/CancerTypeSection/cancerTypeSubtypeStageSideEffectTest.js:12]: revert changes to this file so it doesnt show up

File: cypress/integration/CTS/SearchResultsPageTests.feature [hunk 4]
```diff
@@ -65,19 +65,19 @@ Feature: As an user, I should be able to see my search results displayed, select
...
+#     When user selects 1 result's checkbox
+#     And user clicks on "Print Selected" button at "top" position
+#     Then user is redirected to "/CTS.Print/Display" with generated "printid"
+#     And page title is "Your Clinical Trials Search Results"
+#     And print page search criteria table displays the following
+#         | Category          | Selection     |
+#         | Keywords/Phrases: | breast cancer |
+#     And "New Search" link has a href "/about-cancer/treatment/clinical-trials/advanced-search"
+#     And "Print" link has a href "javascript:window.print();"
+#     And "E-Mail These Results" link has a "target" attribute with "_blank" value
```
Comment (kate-mashkina) [inline cypress/integration/CTS/SearchResultsPageTests.feature:81]: same - revert changes here so it doesnt show up

File: cypress/integration/CTS/TrialDescriptionGeneralPageComponentsTest.feature [hunk 5]
```diff
@@ -2,6 +2,8 @@ Feature: As an user, I should be able to see trial description page and all it's
 
     Scenario: Trial description page displays general page components
         Given user is navigating to "/about-cancer/treatment/clinical-trials/search/r?loc=0&rl=2"
+        And browser waits
```
Comment (kate-mashkina) [inline cypress/integration/CTS/TrialDescriptionGeneralPageComponentsTest.feature:5]: remove waits

File: cypress/integration/CTS/TrialDescriptionGeneralPageComponentsTest.feature [hunk 6]
```diff
@@ -27,6 +29,8 @@ Feature: As an user, I should be able to see trial description page and all it's
     Scenario: Trial description page displays general page components on mobile
         Given screen breakpoint is set to "mobile"
         Given user is navigating to "/about-cancer/treatment/clinical-trials/search/r?loc=0&q=breast%20cancer&rl=2"
+        And browser waits
+        And browser waits
```
Comment (kate-mashkina) [inline cypress/integration/CTS/TrialDescriptionGeneralPageComponentsTest.feature:33]: remove waits

**Review by KateMashkinaNIH — APPROVED** [review]
